### PR TITLE
[LA-87] Add private git repo support for eb extensions cloud-lrs deployment

### PR DESCRIPTION
**Pull request by sandeep-jay**
Added github SSH keys configuration under ebextensions for node environment to resolve private package dependencies. The SSH keys are made available under an S3 bucket requiring authentication. The git configurations for Elastic Beanstalk downloads the keys and adds it to the .ssh folder. All of these tasks need to be performed as a root user (Not as ec-user or node user).

Also, migrated cloud-lrs to node 6.10.0 and updated package.json to mitigate npm install errors on elastic beanstalk.

Bunyan was updated to latest version 2.0.0. Bunyan uses dtrace-provider as a dependency which in turn refers nan(2.6.2). The latest version of nan was causing npm install errors. Nan has been downgrade to 2.5.1.

Removed gulp-imagemin, gulp-append-data, gulp-consolidate packages as LRS is not using them. imagemin requires a dependency called gifsicle(3.0.4) which results in npm install errors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,15 +15,15 @@
   "dependencies": {
     "async": "2.1.4",
     "basic-auth": "1.1.0",
-    "big-integer": "latest",
+    "big-integer": "1.6.23",
     "body-parser": "1.17.1",
     "bower": "1.8.0",
     "bunyan": "1.8.10",
     "bunyan-prettystream": "0.1.3",
+    "caliperjs": "git@github.com:sandeepmjay/caliper-js.git#develop",
```
Comment (sandeep-jay) [inline package.json:23]: This will be replaced with IMS Global develop version once updates are made to Caliper ingestion functions in LRS.

**Comment by pauline2k** [comment]
Looks good, @sandeepmjay.

**Comment by sandeep-jay** [comment]
Thanks @kerschen for the review !

File: package.json (hunk 1 above)
Comment (raydavis) [reply package.json:23]: @sandeepmjay , this blocks my ability to "npm install" the master branch. If you aren't allowed to give collaborators the right to fetch from the private repo, I'm afraid we'll have to continue with a documented manual step.

Comment (sandeep-jay) [reply package.json:23]: @raydavis : Agreed. For now, we will need to manually add the package for local deployments. For AWS deployments I have made ssh keys available on S3 which should resolve things automatically. Also, IMS will be making a public release by end of the month for caliper-js.

**Pull request merged**